detox/src/devices/drivers/android/genycloud/GenyCloudDriver.js

```diff
@@ -165,4 +167,10 @@ function reportGlobalCleanupSummary(deletionLeaks) {
   }
 }
 
+function rawDevicesToInstanceHandles(rawDevices) {
```

Comment:
So, basically, you still have to dance with a tambourine here, to not rewrite further code in this PR?

Reply:
LOL it's mostly for clarity. Writing further code boils down to #2193, doesn't it?